### PR TITLE
add basic depths with stairs and bumper to access it

### PR DESCRIPTION
**Pull request by TrueBabyChaise**
Generate map with depths or no with stairs and bumper to access it.

## Information

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Non-breaking changes

## Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation

### CODE REVIEW

**Review by Arthi-chaud — APPROVED** [review]

**Review by Octopus773 — APPROVED** [review]

File: sources/Map/Map.cpp [hunk 1]
```diff
@@ -145,36 +196,46 @@ namespace BBM
 
 	MapGenerator::BlockType MapGenerator::getRandomBlockType()
 	{	
-		double rnd = static_cast<double>(std::rand())/RAND_MAX;
+    	double rnd = static_cast<double>(std::rand())/RAND_MAX;
```
Comment (AnonymusRaccoon) [inline sources/Map/Map.cpp:199]: Indent

File: sources/Map/Map.cpp [hunk 2]
```diff
@@ -145,36 +196,46 @@ namespace BBM
...
 		
 		if (rnd > 0.95)
 			return HOLE;
-		if (rnd > 0.10)
+		if (rnd > 0.25)
 			return BREAKABLE;
-		return NOTHING;
+    	return NOTHING;
```
Comment (AnonymusRaccoon) [inline sources/Map/Map.cpp:205]: Indent

File: sources/Map/Map.cpp [hunk 3]
```diff
@@ -9,52 +9,72 @@ namespace RAY3D = RAY::Drawables::Drawables3D;
...
 				}
 			}
 		}
 	}
 
 	void MapGenerator::generateWall(int width, int height, std::shared_ptr<WAL::Scene> scene)
 	{
-		std::string unbreakableObj = "assets/wall/unbreakable_wall.obj";
-		std::string unbreakablePnj = "assets/wall/unbreakable_wall.png";
+		static const std::string UnbreakableObj = unbreakableWallPath + objExtension;
```
Comment (AnonymusRaccoon) [inline sources/Map/Map.cpp:43]: Keep your naming consistent

File: sources/Map/Map.cpp (hunk 1 above)
Comment (Octopus773) [reply sources/Map/Map.cpp]: https://github.com/AnonymusRaccoon/Bomberman/pull/99/commits/d6992701df94fc6223904486a70965c27e72efdf

File: sources/Map/Map.cpp (hunk 2 above)
Comment (Octopus773) [reply sources/Map/Map.cpp]: https://github.com/AnonymusRaccoon/Bomberman/pull/99/commits/d6992701df94fc6223904486a70965c27e72efdf

File: sources/Map/Map.cpp (hunk 3 above)
Comment (Octopus773) [reply sources/Map/Map.cpp]: https://github.com/AnonymusRaccoon/Bomberman/pull/99/commits/96713ba549ce1950249529270b777fc9bdfb1349

**Pull request merged**